test/cases/040_packages/008_format_mount/001_by_label/test.yml

```diff
@@ -0,0 +1,30 @@
+kernel:
+  image: linuxkit/kernel:4.9.38
+  cmdline: "console=ttyS0 page_poison=1"
+init:
+  - linuxkit/init:059b2bb4b6efa5c58cf53fed4d0ea863521959fc
+  - linuxkit/runc:4a35484aa6f90a1f06cdf1fb36f7056926a084b9
+onboot:
+  - name: format
+    image: linuxkit/format:efafddf9bc6165b5efaf09c532c15a1100a10e61
+    command: ["/usr/bin/format", "-label", "docker", "/dev/sda"]
```

Comment:
This still hard codes the `/dev/sda` so presumably fails when executed on hyperkit